apps/front/src/app/(auth)/register/page.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import RegisterForm from '@src/app/(auth)/register/registerForm/registerForm'
+import Image from 'next/image'
+import SignUpPhoto from '../../../../public/assets/signUpPhoto.png'
+import Logo from '../../../../public/assets/logo.png'
+import Link from 'next/link'
+
+const Register = () => {
+    return (
+        <div className='min-h-screen text-white font-raleway flex justify-center items-center bg-black'>
+            <Image
+                src={Logo}
+                alt='Kanban logo'
+                className='absolute z-10 left-1 top-1'
+                width={60}
+                quality={100}
+            />
+            <div className='md:w-3/4 m-5 bg-gradient-to-br from-[#00dffc] flex to-[#00ff82] items-center rounded-2xl border-[rgba(255,255,255,0)] [box-shadow:0px_0px_31px_9px_rgba(0,_0,_0,_1)] relative'>
+                <div className='hidden md:flex justify-center items-start w-1/3'>
+                    <div className='text-4xl text-black text-start absolute top-10 m-10'>
+                        Memorize your tasks with Kanban
```

Comment:
![Uploading Screenshot 2023-12-04 at 19.56.20.png…]()

Reply:
Damnienieczka i GPT mówią że jest to napisane poprawnie pod względem gramatycznym.